ui/src/app/machines/views/MachineDetails/MachineSummary/OverviewCard/StatusCard/StatusCard.tsx

```diff
@@ -48,6 +48,12 @@ const StatusCard = ({ machine }: Props): JSX.Element => {
             {formattedOS}
           </p>
         ) : null}
+        {machine.error_description &&
+        machine.status_code === NodeStatusCode.BROKEN ? (
+          <p className="u-text--muted" data-test="error-description">
```

Comment:
There's no max length for `machine.error_description` so I think we might want to set one manually here, otherwise the card gets all kinds of wonky with a long description. Maybe we can break it out into a tooltip or have a "Show more..." button?

Reply:
How about if it truncates the line just like the machine list?

<img width="450" alt="Screen Shot 2021-04-22 at 3 43 17 pm" src="https://user-images.githubusercontent.com/361637/115661932-958bae00-a381-11eb-9291-64c65de9956d.png">

Reply:
@amylily1011 or @lyubomir-popov do you have a suggestion about how we might display this? The message could be any length, so it could be a lot of content to show in the column.

<img width="385" alt="Screen Shot 2021-04-22 at 4 58 06 pm" src="https://user-images.githubusercontent.com/361637/115669961-397a5700-a38c-11eb-958b-c34b4a651e7d.png">

Reply:
HI, a couple of questions to understand the issue better:

- Is this the only piece of information within that layout that doesn't have a character limit?
- Is this a place where people put a lot of information in real world use?

Reply:
> Is this the only piece of information within that layout that doesn't have a character limit?

Yes, everything else is either fixed strings (or one of a choice of fixed strings).

> Is this a place where people put a lot of information in real world use?

That I don't know about. @amylily1011 do you have any idea what kind of info people might use for the mark-broken messages?

Reply:
Here's the full card if that helps:
<img width="1420" alt="Screen Shot 2021-04-22 at 4 56 33 pm" src="https://user-images.githubusercontent.com/361637/115795188-c9fb7a80-a412-11eb-80aa-98a8eea26e07.png">

Reply:
I've moved this discussion here so we can follow up with an improved design: https://github.com/canonical-web-and-design/maas-ui/issues/2565